spec/common/providers/identity.spec.ts

```diff
@@ -762,4 +770,38 @@ describe("identity", () => {
       );
     });
   });
+
+  describe("generateRequestPayload", () => {
+    const DISPLAY_NAME_FILED = "displayName";
+    const TEST_RESPONSE = {
+      displayName: TEST_NAME,
+      recaptchaPassed: false,
+    } as identity.BeforeCreateResponse;
+
+    const EXPECT_PAYLOAD = {
+      userRecord: { displayName: TEST_NAME, updateMask: DISPLAY_NAME_FILED },
+      recaptchaPassed: false,
```

Comment:
nit: can we try a test case where recaptchaPassed is true?

Reply:
added